src/server/utils/response.utils.js

```diff
@@ -3,6 +3,24 @@ const { VERSIONS } = require('../../constants');
 const errors = require('./error.utils');
 const path = require('path');
 
+const defaultOperation = {
```

Comment:
Could you make this a function that takes in optional 'id' and 'message' arguments and returns an operation outcome? This way others could supply their own ids/text to suit their needs. If not supplied, these options can default to allok.

Reply:
Will do asap

Reply:
So I can't figure any way to get the OperationOutcome to produce any useful information based on whats available from the controller. I've created a function that uses these defaults. Perhaps that can be used in the next version somehow.